### ISSUE TITLE
Combine files in debug

### ISSUE THREAD

**Issue by bmarek-ias** [kriswallsmith/assetic#844]
I have a particulate issue where I need assets to be combined in debug, but I need "debug: '%kernel.debug%'" because mimifier to work in prod but not in dev. How do I configure assetic to do that?

**Comment by bmarek-ias** [comment]
I think I got it: combine=true

{% javascripts
'bundles/assets/js/app.config.js'
'bundles/assets/js/app.js'
'bundles/assets/js/notification/SmartNotification.js'
'bundles/assets/js/smartwidgets/jarvis.widget.js'
combine=true
filter='?uglifyjs2'
output='js/compiled_app.js' %}

**Comment by stof** [comment]
yeah, this is indeed what you need. The default value for ``combine`` is based on the ``debug`` mode (inverting it), but you can force combining the assets all the time. (this cannot be done globally in the config though)
